back/lib/participation_method/voting.rb

```diff
@@ -14,6 +14,10 @@ def posting_allowed?
       false
     end
 
+    def supports_reacting?
```

Comment:
Not sure why this and the associated test changes are showing up here as these are already in the i1 branch